.env.sample

```diff
@@ -1,8 +1,280 @@
-POSTGRES_USER=postgres
-POSTGRES_PASSWORD=postgres
-POSTGRES_DB=keygen
-KEYGEN_HOST=api.keygen.localhost
-KEYGEN_EDITION=CE
-KEYGEN_MODE=singleplayer
-KEYGEN_ADMIN_EMAIL=test@test.com
-KEYGEN_ADMIN_PASSWORD=123456
+# ------------------------------------------------------------------------------
+# Environment Variables
```

Comment:
Can you add a link to the self-hosting docs here in case there are additional questions/concerns?